src/App.tsx

```diff
@@ -1,20 +1,17 @@
 import '@assets/app.scss'
-import AppLayout from '@app/layouts/AppLayout.tsx';
+import AppLayout from '@app/layouts/app-layout.tsx';
```

Comment:
C'est une nouvelle pratique d'ajouter des tirets et retirer les majuscules ? :o Je découvre

Reply:
Ah bah alors aucune idée, c'est comme ça sur Rigby et je sais pas si ça vient de nos pratiques ou des leurs !

Reply:
Ah non, ce sont des pratiques propres à Rigby (une préférence de Guillaume par rapport à des soucis de casse sur Windows qu'il a pu rencontrer). On s'est alignés sur le projet, mais je suis pour conserver du PascalCase chez nous 😃

Reply:
Team Pascal et pas kebab, c'est noté 🌮